api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1177,91 +1171,13 @@ public void setDateCreated(Date dateCreated) {
 	}
 	
 	/**
-	 * Gets the namespace for the form field that was used to capture the obs details in the form
-	 * 
-	 * @return the namespace
-	 * @since 1.11
-	 * <strong>Should</strong> return the namespace for a form field that has no path
-	 * <strong>Should</strong> return the correct namespace for a form field with a path
-	 * <strong>Should</strong> return null if the namespace is not specified
-	 */
-	public String getFormFieldNamespace() {
```

Comment:
Instead of removing, shouldn't we deprecate these public methods? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
@dkayiwa this method doesn't really get removed from `Obs`, it's now coming from a new superclass.

Reply:
Oh cool 😊

Reply:
@dkayiwa here: https://github.com/openmrs/openmrs-core/pull/3357/files#diff-26312f3ab88527f26370de68fc508603R64